### PR TITLE
add thejaswin123

### LINKED ISSUE #507: Thejaswin S
screencapture-github-thejaswin123-2021-05-14-22_36_28

### PR DESCRIPTION
**Pull request by thejaswin123**
resolve #507

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -892,5 +892,12 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/60679101/118286430-d1112800-b4a8-11eb-8bc9-9152ba87d351.png",
     "issueNumber": 501
-  }
+  },
+  {
+    "name": "Thejaswin S",
+    "githubUsername": "thejaswin123",
+    "imageUrl": "https://user-images.githubusercontent.com/52855622/118305007-08152700-b505-11eb-919e-268ffa68a0ff.png",
+    "issueNumber": 507
+}
```
Comment (starlightknown) [inline data.json:901]:
```suggestion
    }
```

File: data.json [hunk 2]
```diff
@@ -892,5 +892,12 @@
...
+
```
Comment (starlightknown) [inline data.json:902]:
```suggestion

```

**Review by starlightknown — CHANGES_REQUESTED** [review]
remove the extra line and align the bracket

**Review by schmelto — CHANGES_REQUESTED** [review]
Please resolve the issues / suggestions from @starlightknown otherwise it looks good for me

**Comment by thejaswin123** [comment]
> remove the extra line and align the bracket

removed the extra lines and aligned the brackets

**Comment by thejaswin123** [comment]
> Please resolve the issues / suggestions from @starlightknown otherwise it looks good for me

resolved

**Review by starlightknown — APPROVED** [review]
LGTM 👍🏼

**Review by deephunt3r — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
Thanks @starlightknown and @deephunt3r for reviewing 🚀

**Pull request merged**